types/prismic-dom/index.d.ts

```diff
@@ -15,6 +53,7 @@ interface Link {
 
 export const RichText: RichText;
 export const Link: Link;
+export const HTMLSerializer: HTMLSerializer<string>;
 
-declare const _default: { RichText: RichText, Link: Link };
+declare const _default: {RichText: RichText; Link: Link};
```

Comment:
why is this a semicolon now?

Reply:
Just trying to be more consistent with other packages (e.g. [node-xlsx](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node-xlsx/index.d.ts#L31-L35))

Indentation was updated